src/frontend/tsx/Games.tsx

```diff
@@ -22,6 +22,15 @@ export function Games () {
     await pokeAction(urbit, offerDraw(gameID), null, () => { offeredDraw(gameID) })
   }
 
+  const [checked, setChecked] = useState(false)
```

Comment:
1. Need a better name than just "checked". What this value really represents is the status of the user's preference for auto-claiming special draws, so it should reflect that.
2. There's a bug here right now, since this state is enabled/disabled per-game, but the state is represented by a single boolean. What we would need is a `set` in the frontend called something like "auto-claim-games", and we would add/remove games to/from it when the user preference changes. When rendering the checkbox, we would check for the current game ID in this set to know whether or not it should be rendered as checked.
3. It's important to note that for ultimate correctness, the frontend should add/remove the game from the set of games with auto-claim enabled *based on an update from the backend, not on click in the checkbox*. This is because the backend state in Urbit is the ultimate source of truth, and we don't want the frontend to get out of sync.

The above means a flow that looks something like this:
1. New game starts with default preference for auto-claiming special draws
2. Game info sent to frontend; add game ID to set of games with auto-claim enabled if necessary
3. When rendering game, compute checkbox state by checking if game ID in set
4. Checkbox function = send auto-claim preference poke with opposite of current setting
5. Backend receives this poke, updates state, and sends update to frontend with new preference
6. Frontend receives update, adds/removes game ID to/from set of set of games with auto-claim enabled

This means adding another member & function to `chessState`/`chessStore`, and it'll also mean adding another update type to the backend.

Reply:
Rather than the `set` of auto-claim-games, wouldn't it be more in line with how we currently store this kind of data to add `autoClaimSpecialDraws` to `ActiveGameInfo`?

Reply:
You're right, that would be better. In that case, the the frontend will need to update the state of automatic claims for the game independently. This isn't ideal, but we're already doing it with `receiveUpdate`, so we may as well be consistent.